src/Raven.Server/Documents/PeriodicBackup/RavenStorageClient.cs

```diff
@@ -9,17 +9,35 @@
 using System.Diagnostics;
 using System.IO;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace Raven.Server.Documents.PeriodicBackup
 {
-    public abstract class RavenStorageClient : IDisposable
+    public abstract class RavenStorageClient
     {
-        private readonly List<HttpClient> _clients = new List<HttpClient>();
+        protected static readonly HttpClient HttpClient;
+
         protected readonly CancellationToken CancellationToken;
         protected readonly Progress Progress;
         protected const int MaxRetriesForMultiPartUpload = 5;
 
+        static RavenStorageClient()
+        {
+            var handler = new HttpClientHandler
+            {
+                AutomaticDecompression = System.Net.DecompressionMethods.None
+            };
+
+            var client = new HttpClient(handler)
+            {
+                Timeout = TimeSpan.FromHours(24)
+            };
+
+            HttpClient = client;
```

Comment:
Using a read-only HttpClient might be a problem, since it will fail to handle DNS changes
https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests

(We address this issue in the client as well by replacing the HttpClient upon some failures)

Reply:
We are not using MS DI, so using HttpClientFactory might not be a good idea. But we can use one HttpClient per StorageClient instance. Should be more than enough.